src/node_quic_stream.h

```diff
@@ -101,12 +104,14 @@ class QuicStream : public AsyncWrap,
   int ReadStop() override;
 
   // Required for StreamBase
-  int DoShutdown(ShutdownWrap* req_wrap) override;
+  int DoShutdown(
```

Comment:
This can be in single line, as `SendPendingData` method.

Reply:
Planning to leave these as they are for now. Will fix up styles in a future iteration